paymentsheet/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -186,7 +186,7 @@ internal class SignUpViewModel @Inject constructor(
 
         when (signupResult) {
             is LinkAuthResult.AttestationFailed -> {
-                moveToWeb()
+                moveToWeb(signupResult.error)
```

Comment:
This is not really needed as part of the fix, but gives visibility on the attestation error that caused falling back to web.